.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+# Checklist:
+
+- [ ] I have mentioned the appropriate tag (either of feat, docs, fix) in the commit message and PR title for semantic checks to pass.
```

Comment:
Would be good to add some information abt how to write the commit message here: https://jenkins-x.io/community/documentation/workflow/#commit-and-push-your-changes and link this doc to the PR template. For now, we can link this: https://jenkins-x.io/community/code/#the-commit-message

Also change tag to type: https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commit-message-format -> `type(scope): subject`

Reply:
Done